contracts/Strategy.sol

```diff
@@ -855,7 +854,7 @@ contract Strategy is BaseStrategy {
 
     // Effective collateralization ratio of the strat
     function getCurrentCollateralRatio() public view returns (uint256 _collateralRate) {
-        if (balanceOfDebt() == 0) return C_RATE_PRECISION.div(MAX_BPS);//dev: this should represent infinity in this case
+        if (balanceOfDebt() == 0) return maxCollaterizationRate.add(rebalanceTolerance*5);//dev: this should represent infinity in this case
```

Comment:
@newmickymousse any particular reason why the rebalancing tolerance is added five times?

Reply:
Given that we cannot divide by 0, we need to represent something big enough so the strategy can operate automatically. For example, `balanceDebt() == 0` means we can borrow from abra. `adjustPosition` will check that `currentCollateralRatio` is larger than `maxCollaterizationRate` and will mint more mim